datafusion/core/tests/parquet/page_pruning.rs

```diff
@@ -108,14 +108,16 @@ async fn page_index_filter_one_col() {
 
     let batch = results.next().await.unwrap().unwrap();
 
-    // `month = 12` from the page index should create below RowSelection
-    //  vec.push(RowSelector::skip(894));
-    //  vec.push(RowSelector::select(339));
-    //  vec.push(RowSelector::skip(3330));
+    // `month = 1` or `month = 2` from the page index should create below RowSelection
```

Comment:
These tests were just wrong, I have fixed them using https://github.com/apache/arrow-rs/pull/3405 to verify that they are correct